### ISSUE TITLE
Accuracy on multi gpus with nn.DataParallel

### ISSUE THREAD

**Issue by ShaggyBlanco** [erikwijmans/Pointnet2_PyTorch#88]
Hello, @erikwijmans. I'm currently training my own custom point cloud dataset (consisting of geometric features X, Y, Z and normals Nx, Ny, Nz) on a p3.x16large AWS EC2 instance using all eight Nvidia Tesla V100 GPUs (16GB GPU memory each, with a total GPU memory of 128GB). To test out the advantage of data parallelism using nn.DataParallel, I've chosen a batch size of 256 which are split across eight GPUs so that each of them can process a batch of 32. Other hyperparameters are set to the following values:

bn_momentum: 0.5 
bnm_decay: 0.5 
checkpoint: None, 
decay_step: 8000.0, 
epochs: 300, 
learning rate: 0.00001, 
lr_decay: 0.7, 
num_points: 1024,  
weight_decay: 0.0001
train_test_ratio: 70%:30% 

Seeing from the results, the overall accuracy from validation (around 82.52%) seems not to increase dramatically compared to the one from a local workstation using single Quadro RTX 6000 GPU (24GB GPU Memory) and trained with a batch size of 32, applying the same hyperparameter settings mentioned above (around 83%). Am I missing something here (e.g. suboptimal choice of hyperparameters, or further modification of model layers needed)? Thank you in advance.

P.S.: I'm still new to deep learning stuff. Please correct me if I misunderstand something in my question :)

**Comment by erikwijmans** [comment]
An LR of `1e-5` seems low, but Adam generally does a good job of dealing with low learning rate so that probably doesn't matter. 

Using DataParallel will decreasing training time, but I have never head of it increases model performance. In general, large batch training _decreases_ performance in supervised.  I recommend reading these papers https://arxiv.org/abs/1609.04836 https://arxiv.org/abs/1706.02677
